js/main.js

```diff
@@ -83,7 +124,7 @@ function renderEntry(object) {
 }
 
 function contentLoaded(event) {
-  for (var i = 0; i < data.entries.length; i++) {
+  for (i = 0; i < data.entries.length; i++) {
```

Comment:
you should definitely declare your variable here instead of using a global.

Reply:
Done.